kpops/component_handlers/topic/model.py

```diff
@@ -29,9 +26,6 @@ class TopicResponse(BaseModel):
         # https://github.com/confluentinc/kafka-rest/issues/1085
     )
 
-    class ModelConfigs(BaseConfig):
-        extra = Extra.forbid
-
```

Comment:
this never worked because the class was named correctly (should have been `Config`). when fixing the mistake the dry-run fails, because the model is incomplete, e.g. missing `authorized_operations`. I believe we can just ignore the extra fields which is the default behavior anyways